pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1573,6 +1575,8 @@ public void addTable(TableConfig tableConfig)
           + "If the external view is not removed after a long time, try restarting the servers showing up in the "
           + "external view");
     }
+
+    LOGGER.info("Adding table {}: Validate table tenant config", tableNameWithType);
```

Comment:
Let's rephrase this as `Validate table configs`

Also, let's add one line before 1594: `Creating table config in the property store`

Reply:
Addressed the comments